mlfinlab/structural_breaks/sadf.py

```diff
@@ -26,6 +28,8 @@ def _get_sadf_at_t(X: pd.DataFrame, y: pd.DataFrame, min_length: int) -> float:
         if not np.isnan(b_mean_[0]):
             b_mean_, b_std_ = b_mean_[0, 0], b_std_[0, 0] ** 0.5
             all_adf = b_mean_ / b_std_
+            if model[:2] == 'sm':
+                all_adf = np.abs(all_adf) / (y.shape[0]**phi)
```

Comment:
Should this be an elif statement? The next line.

Reply:
No, this regularization is done only in the case of SM test. The update procedure is the same: we take the maximum of all ADF statistic in the backwards window